TMTumblrSDK/Core/TMSDKFunctions.h

```diff
@@ -6,6 +6,7 @@
 //  Copyright (c) 2013 Tumblr. All rights reserved.
 //
 
+@import Foundation;
```

Comment:
Fixed the dupe imports.

`@import` shouldn't break, because Apple frameworks (Foundation, UIKit etc.) are always available as modules. `pod install` worked fine when I tested it.